ui/public/index.html

```diff
@@ -17,6 +17,9 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js" integrity="sha512-s+xg36jbIujB2S2VKfpGmlC3T5V2TF3lY48DX7u2r9XzGzgPsa6wTpOQA7J9iffvdeBN0q9tKzRxVxw1JviZPg==" crossorigin="anonymous"></script>
 	<script defer src="/build/bundle.js"></script>
 
+	<!-- Fontawesome-->
+	<script src="https://kit.fontawesome.com/551d07ef8f.js" crossorigin="anonymous"></script>
```

Comment:
see comment below, we don't wnt to add whole new library just to use 1 single icon, use the same SVG icon library that we use